src/sql/run.py

```diff
@@ -469,7 +486,15 @@ def run(conn, sql, config):
         else:
             txt = sqlalchemy.sql.text(statement)
             manual_commit = set_autocommit(conn, config)
-            result = conn.session.execute(txt)
+
+            is_custom_connection = Connection.is_custom_connection(conn)
+            if is_custom_connection:
+                txt_ = str(txt)
+            else:
+                txt_ = txt
+            # stringify txt to avoid TypeError:
+            # Boolean value of this clause is not defined
+            result = conn.session.execute(txt_)
```

Comment:
Is it better to use `sqlalchemy.text`?
```
from sqlalchemy import text
```
```suggestion
            result = conn.session.execute(text(txt_))
```

Reply:
sqlalchemy.sql.text returns TextClause which can be used as an argument to an SQLAlchemy query but it won't work when we'll use a custom connection.